app.yaml

```diff
@@ -19,6 +19,7 @@ handlers:
   login: admin
 - url: /.*
   script: _go_app
+  login: required
```

Comment:
This is probably OK for the current pass, but I think I will want people to be able to browse the site without having to be logged in.